chapters/ranker.rst

```diff
@@ -0,0 +1,155 @@
+==========================================
+A Guide to Jina Ranker
+==========================================
+
+.. meta::
+   :description: A guide on Jina Ranker
+   :keywords: Jina, Ranker
+
+.. note:: This guide assumes you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://101.jina.ai>`_ first.
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Motivation
+--------------------
+
+In a searching system, query accuracy is one of the most important aspects that users will concern. In most cases, we expect well-ordered query results to be presented.
+For one search request, we may have multiple matches and we want to have the most relevant match or probably top-K matches presented to users. On the other hand, we may also want to aggregate the top-K chunks to top-K matches.
+
+
+Before you start
+-------------------
+
+We expect you have a clean Python 3.7/3.8/3.9 (virtual) environment.
+Install Jina on your machine:
+
+.. highlight:: bash
+.. code-block:: bash
+
+    pip install -U jina
+
+
+Overview
+-----------------
+
+To achieve our objectives, Jina has a family of :term:`Executor` named :term:`Ranker`.
+Jina :term:`Ranker` inherits either the :class:`Match2DocRanker` or the :class:`Chunk2DocRanker`.
+For the :class:`Match2DocRanker`, it re-scores the matches with different mechanisms. For the :class:`Chunk2DocRanker`, it aggegrates the chunks to documents.
+
+
+Match2DocRanker
+^^^^^^^^^^^^^^^^
+
+:class:`Match2DocRanker` re-scores the matches for a document. This Ranker is only responsible for calculating new scores and not for the actual sorting. The sorting is handled in the respective :class:`Matches2DocRankDriver`.
+
+
+Chunk2DocRanker
+^^^^^^^^^^^^^^^
+
+:class:`Chunk2DocRanker` translates the chunk-wise score to the doc-wise score. If you want to aggregate the already existed top-k chunks into top-k  documents, :class:`Chunk2DocRanker` will be your choice.
+
+
+Drivers
+^^^^^^^^^
+Different :term:`Rankers` will need different :term:`Drivers` to be equipped with.
+For :class:`Match2DocRanker`, we need to choose :class:`Matches2DocRankDriver` to get the scores from a :term:`Ranker` and help to resort the scores.
+For :class:`Chunk2DocRanker`, we need to choose :class:`Chunk2DocRankDriver` to extract matches score from chunks and use the executor to compute the rank and assign the resulting matches to the level above.
+
+Currently, Jina has 3 Rankers in Jina hub.
```

Comment:
three. 

My rule is any number less than ten should be written in text, for numbers above ten you can use numbers. 

One, 
Eight, 
10201

Reply:
Why is that?

Reply:
"In scientific and technical writing, the prevailing style is to write out numbers under ten. "[[Reference](https://www.grammarly.com/blog/when-to-spell-out-numbers/?gclid=Cj0KCQiAj9iBBhCJARIsAE9qRtAeV-59uHLIKyGCmSKOvAfGt-1ckY6UCtq4VfZcvI0RzFfujeg9xcQaAgTDEALw_wcB&gclsrc=aw.ds)] 

Don't know **why**, it's just the style :)

Reply:
Sure! I will keep that in mind. 👍